SUPPORTED_PLATFORMS.md

```diff
@@ -0,0 +1,90 @@
+# Supported Platforms
+
+This document outlines the platform support matrix for different versions of the Microsoft build of Go.
```

Comment:
I think the meaning of "support" in this context needs to be clarified.

Reply:
updated